packages/expo-ads-admob/android/build.gradle

```diff
@@ -52,7 +52,7 @@ if (new File(rootProject.projectDir.parentFile, 'package.json').exists()) {
   apply from: project(":unimodules-core").file("../unimodules-core.gradle")
 } else {
   throw new GradleException(
-      "'unimodules-core.gradle' was not found in the usual Flutter or React Native dependency locations. " +
+      "'unimodules-core.gradle' was not found in the usual React Native dependency location. " +
```

Comment:
```suggestion
      "'unimodules-core.gradle' was not found in the usual React Android dependency location. " +
```

🤔

Reply:
It's a tough one to reason about, but as I can see it, `unimodules-core` is like a _React Native dependency_, not _React Android dependency_ and we are looking for it in a place usual for a _React Native project_, not a _React Android project_. In general I think _React Android_ is a term more commonly known among RN contributors than developers.

Reply:
I'll let myself merge this PR, but if you feel strongly about this suggestion let's come back to this discussion online!